neural_structured_learning/examples/notebooks/Adversarial_regularization_for_image_classification save and load the adv model.ipynb

```diff
@@ -0,0 +1,710 @@
+{
```

Comment:
<p>How about adding some evaluation to demonstrate that the loaded model performs the same as the model before saving?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/neural-structured-learning/pull/114/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>